2-mandatory/2-retrying-random-numbers.js

```diff
@@ -10,7 +10,13 @@ function generateRandomNumber() {
 }
 
 function getRandomNumberGreaterThan50() {
-    // TODO - implement using a do-while loop
+    let a;
+
+    do {
+        a = generateRandomNumber();
```

Comment:
Nice. What would happen if you instead initiated "a" within the do loop with : let a =generateRandomNumber();? Why would you get an error?